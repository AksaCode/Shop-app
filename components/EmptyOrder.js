import React from 'react';
import { Text, View, StyleSheet, Image } from 'react-native';

import Colors from '../constants/Colors';

const EmptyOrder = (props) => {
  return (
    <View>
      <Text style={styles.mainStyle}>{props.output}</Text>
      <View>
        <Image
          style={{ width: 100, height: 100, marginHorizontal: 140, marginVertical: 40 }}
          source={{
            uri: 'https://www.kaphstore.com/resources/assets/front/images/cartempty.png',
          }}
        />
        {props.children}
      </View>
    </View>
  );
};

const styles = StyleSheet.create({
  mainStyle: {
    textAlign: 'center',
    fontSize: 25,
    fontWeight: '200',
    marginTop: 30,
    color: 'grey',
    borderBottomWidth: 1,
    borderBottomColor: Colors.accentColor,
  },
  image: {
    marginTop: 15,
  },
});

export default EmptyOrder;
