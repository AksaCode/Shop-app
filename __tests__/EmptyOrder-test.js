import React from 'react';
import renderer from 'react-test-renderer';
import EmptyOrder from '../components/EmptyOrder';

test('renders correctly', () => {
  const tree = renderer.create(<EmptyOrder />).toJSON();
  expect(tree).toMatchSnapshot();
});
