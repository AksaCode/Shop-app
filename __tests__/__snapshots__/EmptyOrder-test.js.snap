// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "borderBottomColor": "#191970",
        "borderBottomWidth": 1,
        "color": "grey",
        "fontSize": 25,
        "fontWeight": "200",
        "marginTop": 30,
        "textAlign": "center",
      }
    }
  />
  <View>
    <Image
      source={
        Object {
          "uri": "https://www.kaphstore.com/resources/assets/front/images/cartempty.png",
        }
      }
      style={
        Object {
          "height": 100,
          "marginHorizontal": 140,
          "marginVertical": 40,
          "width": 100,
        }
      }
    />
  </View>
</View>
`;
